rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -444,59 +444,21 @@ BtcBlockStoreWithCache getBtcBlockStore() {
     protected PegTxType getTransactionType(BtcTransaction btcTx) {
         Script retiredFederationP2SHScript = provider.getLastRetiredFederationP2SHScript().orElse(null);
 
-        /************************************************************************/
-        /** Special case to migrate funds from an old federation               **/
-        /************************************************************************/
-        if (activations.isActive(ConsensusRule.RSKIP199) && txIsFromOldFederation(btcTx)) {
-            logger.debug("[getTransactionType][btc tx {}] is from the old federation, treated as a migration", btcTx.getHash());
-            return PegTxType.PEGOUT_OR_MIGRATION;
+        Address oldFederationAddress = null;
```

Comment:
Careful, the old federation address stored in the Bridge constants is not the same as the last retired federation

Reply:
I know. The old federation is only being fetched from the bridge constant. To make it easier to follow I moved the old federation logic out of pegUtilsLegacy, one less param for `PegUtilsLegacy.getTransactionType`.